azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_provider.py

```diff
@@ -48,6 +48,8 @@ def _create_mqtt_client(self):
 
         self._mqtt_client = mqtt.Client(self._client_id, False, protocol=mqtt.MQTTv311)
 
+        self._mqtt_client.enable_logger(logging.getLogger("paho"))
```

Comment:
>self._mqtt_client.enable_logger(logging.getLogger("paho")) [](start = 8, length = 58)

while doing this let's keep an eye out to make sure paho doesn't leak credentials in the logs.